libfreerdp/core/fastpath.c

```diff
@@ -262,14 +262,18 @@ static int fastpath_recv_update(rdpFastPath* fastpath, BYTE updateCode, UINT32 s
 	switch (updateCode)
 	{
 		case FASTPATH_UPDATETYPE_ORDERS:
-			if (!fastpath_recv_orders(fastpath, s))
+			if (!fastpath_recv_orders(fastpath, s)) {
+				WLog_DBG(TAG, "FASTPATH_UPDATETYPE_ORDERS - fastpath_recv_orders()");
 				return -1;
+			}
 			break;
 
 		case FASTPATH_UPDATETYPE_BITMAP:
 		case FASTPATH_UPDATETYPE_PALETTE:
-			if (!fastpath_recv_update_common(fastpath, s))
+			if (!fastpath_recv_update_common(fastpath, s)) {
+				WLog_DBG(TAG, "FASTPATH_UPDATETYPE_ORDERS - fastpath_recv_orders()");
```

Comment:
Wouldn't it make sense to change these WLog_DBG to errors? (Also the following once)